front-end/src/app/shared/models/transaction-groups/transaction-group-a.model.spec.ts

```diff
@@ -56,4 +56,9 @@ describe('TransactionGroupA', () => {
     const retval = component.hasCandidateInformationInput();
     expect(retval).toBeFalse();
   });
+
+  it('#hasCandidateCommitteeInput happy path', () => {
+    const retval = component.hasCandidateCommitteeInput();
```

Comment:
"retval" isn't returned meaning it's not a "return value".  Any reason this can't be ```expect(component.hasCandidateCommitteeInput()).toBeFalse();```?

Reply:
ah true - I overload the term to be 'return value' or 'returned value', but that could be confusing.